src/Compilers/CSharp/Portable/Symbols/Source/ParameterHelpers.cs

```diff
@@ -29,10 +29,19 @@ internal static class ParameterHelpers
             int firstDefault = -1;
 
             var builder = ArrayBuilder<ParameterSymbol>.GetInstance();
-            ImmutableArray<ParameterSymbol> parameters;
+            var mustBeLastParameter = (ParameterSyntax)null;
```

Comment:
I prefer keeping consistency with the locals.  Since the local above is 'var', i like the local below being 'var'.  Since the typename is clear, that fits the rules here i believe :)